mozilla_version/gecko.py

```diff
@@ -303,26 +331,31 @@ def _compare(self, other):
             return channel_difference
 
         if self.is_beta and other.is_beta:
-            beta_difference = self.beta_number - other.beta_number
+            beta_difference = cast(int, self.beta_number) - \
```

Comment:
Too bad to see mypy is not smart enough to know `self.beta_number` is already an int because of the check just above :/